### ISSUE TITLE
Difficult to tell if formats are applied

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#3112]
I assume the button that says "insert section break" will insert a horizontal line. After I insert a line, I don't hear any indication from my screen reader as I move through the email text indicating that a line is there. I just hear the "steps to sign up" heading twice. I can only assume one of those is the horizontal line. (JAWS/Chrome)

---

After selecting the various apply buttons, I assume the requested changes are being applied. However, I don't know how to check. After I select an item and apply the bold, italics, and headings voiceover doesn't say whether that item has been changed. When voiceover doesn't announce whether the changes were made, then I am assuming everything's good. (Voiceover/Safari)

---

When I navigate a list that has been applied in the edited text, I am unable to tell what type of list it is, a bulleted list or a numbered list. It would be nice if this information was available. (NVDA/Chrome)

---

I am not sure if "French" was applied to the sentence because my screen reader is still trying to pronounce it according to English rules. Are you adding language tags in the markup when language is specified by the user? If you add a tag to identify French as French, my screen reader will automatically pronounce it as French. (JAWS/Chrome)

---

Sometimes formats are "applied" sometimes they should be "inserted". For example, you "insert" a section break or a variable, but you "apply" bold to a selection range. Sometimes it can be both, depending on the current selection.

**Comment by amazingphilippe** [comment]
PR to announce editor blocks: https://github.com/cds-snc/notification-admin/pull/2669
